openwisp_notifications/tasks.py

```diff
@@ -204,17 +182,9 @@ def ns_organization_created(instance_id):
     for a newly created organization.
     """
     notification_types = NOTIFICATION_TYPES.keys()
-    notification_settings = []
+    organization = Organization.objects.get(id=instance_id)
     for user in User.objects.filter(is_superuser=True):
-        for type in notification_types:
-            notification_settings.append(
-                NotificationSetting(
-                    user_id=user.id, type=type, organization_id=instance_id
-                )
-            )
-    NotificationSetting.objects.bulk_create(
-        notification_settings, ignore_conflicts=True
-    )
```

Comment:
I think this code can be kept, since there will be no existing notification setting for this organization. Hence, bulk_create will be better here.

Reply:
Let's keep the new version, so we don't have to maintain different versions of the same code.